guidelines/faster-review.md

```diff
@@ -0,0 +1,115 @@
+# Faster Code Review Guideline
+
+Kyma is quite a large project with many areas, each has a group of core developers
+who are working hard on developing that area of Kyma. While that means they are
+working hard on new features and enhancements, they also work on doing code
+reviews for PRs coming from different contributors from the community and it is
+crucial that the code review process of new PRs becomes quick and efficient for
+both parties. This guide defines a set of principles and practices that can help
+achieving such a goal.
+
+## 0. Familiarize yourself with project conventions
+
+- [CONTRIBUTING.md](../CONTRIBUTING.md)
+- [Naming conventions](naming.md)
+- [Content strategy](content-guidelines/content-strategy.md)
+- [Content guidelines](content-guidelines)
+- [Coding standards](coding-standards)
+
```

Comment:
Suggest to add a reference to the DoD too (at present is not yet public)

Reply:
If it's not public yet, we shouldn't put it here.